apps/daimo-web/src/utils/platform.ts

```diff
@@ -12,6 +13,7 @@ export function detectPlatform(ua: string): PlatformType {
   return "other";
 }
 
+// necesary for some routes
```

Comment:
I think we can delete downloadMetadata > use `getDownloadMetadata` everywhere

Reply:
I'm not sure how to do that at the moment, there's this file `route.tsx` in the path /download that detectes the platform, I don't think that the hook `useI18n()` can be used, and I didn't want to over complicate the code